uber/templates/preregistration/badge_refund.html

```diff
@@ -0,0 +1,23 @@
+{% set abandon_action, abandon_button, must_keep='delete', "I'm not coming", not attendee.can_abandon_badge and not attendee.can_refund_badge %}
```

Comment:
This is what using to be the "I'm not coming" button. I split it out into an included template because it's very noisy, but it's not actually very complex -- we're just checking for various reasons they may not be able to abandon/refund their badge and adding a helpful tooltip. It looks like this!
![image](https://user-images.githubusercontent.com/7198215/43350823-bff0c85a-91d8-11e8-9e06-cac764308901.png)